src/main/java/carpetextra/mixins/DispenserBlock_fallingBlockMixin.java

```diff
@@ -0,0 +1,63 @@
+package carpetextra.mixins;
+
+import carpetextra.CarpetExtraSettings;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.block.DispenserBlock;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.FallingBlockEntity;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.state.property.Properties;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.world.World;
+import net.minecraft.world.gen.random.AbstractRandom;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(DispenserBlock.class)
+public class DispenserBlock_fallingBlockMixin {
+
+    // This is what happens to the code when a constructor is private and I just want to change the velocity before spawnEntity xD
```

Comment:
Haven't looked at the vanilla code yet, but does it matter for mixin that something is private? Or wdym with this?

Reply:
I was just mentioning that due to the falling block code being in a private method. I had to make this twice as complicated. Not really related to mixins